source/release-notes.txt

```diff
@@ -20,6 +20,34 @@ Release Notes for |bi|
 .. expect and we don't break the links on the downloads page.
 .. (DOCS-9670 for context)
 
+.. _bi-2-14-5:
+
+|bi| 2.14.5
+-----------
+
+*Released September 2, 2022*
+
+Improvements
+~~~~~~~~~~~~
+
+- Uses ``$collStats`` and ``$currentOp``
+  :manual:`aggregation pipeline </core/aggregation-pipeline>` stages
+  instead of corresponding commands.
```

Comment:
[question] Based on [the ticket](https://jira.mongodb.org/browse/BI-2694) for this change I'm wondering if this is an internal change, not user facing? It might be worth calling out for tech review to see if this could be removed from the release notes. I'm not 100% sure myself.

Reply:
Asked the developer for the ticket and also flagging @rychipman -- should we keep this item in Relnotes?

Reply:
I took a look and this is more of an internal change from a command to aggregation, users would not see a difference in behavior.  It can be removed from the release notes.

Reply:
Removed this item. @rychipman ready for your review.